pkg/ignore/ignore_test.go

```diff
@@ -0,0 +1,248 @@
+package ignore
+
+import (
+	"path/filepath"
+	"strings"
+	"testing"
+	//"flag"
+
+	"github.com/openshift/source-to-image/pkg/api"
+	"github.com/openshift/source-to-image/pkg/util"
+	"github.com/golang/glog"
+	"os"
+)
+
+func getLogLevel() (level int) {
+	for level = 5; level >= 0; level-- {
+		if glog.V(glog.Level(level)) == true {
+			break
+		}
+	}
+	return
+}
+
+
+func baseTest(t *testing.T, patterns []string, filesToDel []string, filesToKeep []string) {
+	// the glog output is even occurring if -v is not specfied;
+	// for now, simplly comment out this section when we don't want glog debug
+	/*
+	from := flag.CommandLine
+	if vflag := from.Lookup("v"); vflag != nil {
+		t.Logf("integration_test.go: found -v flag from starter bash script with initial glog level %d\n", getLogLevel())
+
+		// the thing here is that we are looking for the bash -v passed into test-integration.sh (with no value),
+		// but for glog (https://github.com/golang/glog/blob/master/glog.go), one specifies
+		// the logging level with -v=# (i.e. -v=0 or -v=3 or -v=5).
+		// so, for the changes stemming from issue 133, we 'reuse' the bash -v, and set the highest glog level.
+		// (if you look at STI's main.go, and setupGlog, it essentially maps glog's -v to --loglevel for use by the sti command)
+		//NOTE - passing --loglevel or -v=5 into test-integration.sh does not work
+		if getLogLevel() != 5 {
+			vflag.Value.Set("5")
+			t.Logf("integration_test.go: glog level now %d\n", getLogLevel())
+			// FIXME currently glog has only option to redirect output to stderr
+			// the preferred for STI would be to redirect to stdout
+			flag.CommandLine.Set("logtostderr", "true")
+		}
+	}
+        */
+	// end glog enablement
```

Comment:
Interesting, if a general convention, commented out code can have informative value.  But I admit in this case it was just an ease of use thing, in case I wanted to reuse this logic.  I'll remove as  part of next push.